autoload/airline/extensions/tabline/formatters/webdevicons.vim

```diff
@@ -5,8 +5,10 @@
 
 function! airline#extensions#tabline#formatters#webdevicons#format(bufnr, buffers) abort
   " Call original formatter.
+  let prePadding = g:WebDevIconsTabAirLineBeforeGlyphPadding
```

Comment:
I think it would be better to not have these intermediate variables. Just use the global ones directly on line 11 ?

Reply:
That's true.....